dashboard/src/components/AppView/AppView.tsx

```diff
@@ -37,13 +39,12 @@ export interface IAppViewProps {
 }
 
 interface IAppViewState {
-  deployments: { [d: string]: IResource };
+  manifest: IResource[];
+  deployments: { [d: string]: { resource?: IResource; isFetching: boolean } };
```

Comment:
shouldn't isFetching be outside of the struct? I mean, loading means loading all the resources of that type but not loading a single one isn't it?

Or we are planning to load resources individually?

Reply:
My bad, I think that we actually fetch resources individually, although personally I am not sure if it makes sense to have this flag per resource in our case.

Reply:
yes, it's per element right now.